go.mod

```diff
@@ -1,22 +1,24 @@
 module purestorage/fa-openmetrics-exporter
 
-go 1.19
+go 1.20
 
 require (
+	github.com/akamensky/argparse v1.4.0
 	github.com/go-resty/resty/v2 v2.7.0
 	github.com/google/go-cmp v0.5.9
 	github.com/prometheus/client_golang v1.14.0
 	github.com/prometheus/client_model v0.3.0
+	gopkg.in/yaml.v3 v3.0.1
 )
 
 require (
 	github.com/beorn7/perks v1.0.1 // indirect
-	github.com/cespare/xxhash/v2 v2.2.0 // indirect
+	github.com/cespare/xxhash/v2 v2.1.2 // indirect
 	github.com/golang/protobuf v1.5.2 // indirect
-	github.com/matttproud/golang_protobuf_extensions v1.0.4 // indirect
-	github.com/prometheus/common v0.39.0 // indirect
-	github.com/prometheus/procfs v0.9.0 // indirect
-	golang.org/x/net v0.5.0 // indirect
-	golang.org/x/sys v0.4.0 // indirect
```

Comment:
Any reason we are downgrading some dependencies?

Reply:
It is the go lamg compiler that resolves all the dependencies this way automatically. It should discover the minimal dependency tree, therefore the reuldting go.mod should contain the right version for each of the modules.